src/components/screens/SendScreen/components/NumericKeyboard.tsx

```diff
@@ -0,0 +1,59 @@
+import Icon from "components/sds/Icon";
+import { Display } from "components/sds/Typography";
+import useColors from "hooks/useColors";
+import React from "react";
+import { View } from "react-native";
+
+interface NumericKeyboardProps {
+  onPress: (value: string) => void;
+}
+
+const NumericKeyboard: React.FC<NumericKeyboardProps> = ({ onPress }) => {
+  const { themeColors } = useColors();
+  
+  const renderButton = (content: string | React.ReactNode) => (
+    <View className="flex-1">
+      <View
+        className="h-[48px] items-center justify-center rounded-[12px]"
+        onTouchEnd={() => onPress(typeof content === "string" ? content : "")}
+      >
+        {typeof content === "string" ? (
+          <Display xs medium>
+            {content}
+          </Display>
+        ) : (
+          content
+        )}
+      </View>
+    </View>
+  );
+
+  return (
+    <View className="gap-[12px]">
+      <View className="flex-row gap-[12px]">
+        {renderButton("1")}
+        {renderButton("2")}
+        {renderButton("3")}
+      </View>
+      <View className="flex-row gap-[12px]">
+        {renderButton("4")}
+        {renderButton("5")}
+        {renderButton("6")}
+      </View>
+      <View className="flex-row gap-[12px]">
+        {renderButton("7")}
+        {renderButton("8")}
+        {renderButton("9")}
+      </View>
+      <View className="flex-row gap-[12px]">
+        {renderButton(".")}
```

Comment:
considering the way this NumericKeyboard functionality has been built it appears the "." button is useless, could you confirm that? should we remove this button instead?

another thing to have in mind, is that this keyboard is hardcoded to support only "american" formatted numbers which means it only uses dot (".") and can't use comma (",") or any other localized number separator. I think this is fine for now as we are not aiming to support other languages soon but we should revisit this once we start planning on expanding the app to other languages

Reply:
ah, I see you are using `formatFiatAmount()` when displaying the amounts, that should probably suffice for the localization issue I mentioned above :)